.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,7 +2,7 @@
 - [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
 - [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
 - [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
-- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?
+- [ ] Does your manifest conform to the [1.1 schema](https://github.com/denelon/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?
```

Comment:
Shouldn't this be
```suggestion
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?
```
? 🤔

Reply:
Nvm didn't get to https://github.com/microsoft/winget-pkgs/pull/56407/files#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35L5-R5

Reply:
I fixed it in the other PR :)